datafusion/core/src/physical_plan/planner.rs

```diff
@@ -574,11 +569,12 @@ impl DefaultPhysicalPlanner {
 
                     // TODO: dictionary type not yet supported in Hash Repartition
                     let contains_dict = groups
+                        .expr
```

Comment:
I think it is a minor thing, but one might imagine keeping the fields of `PhysicalGroupBy` private and adding functions like `fn expr()` and `fn is_empty()` mostly as a way of additional documentation

Reply:
Yeah, I think that is a food idea. Fixed.